parserator_web/templates/parserator_web/index.html

```diff
@@ -38,5 +38,49 @@ <h4>Parsing results</h4>
 {% endblock %}
 
 {% block extra_js %}
-  <script src="{% static 'js/index.js' %}"></script>
+  <script>
+    document.addEventListener('DOMContentLoaded', function() {
+      document.querySelector('form').addEventListener('submit', function(event) {
+        event.preventDefault();
+
+      const addressInput = document.getElementById('address').value;
+      
+      fetch(`/api/parse/?address=${encodeURIComponent(addressInput)}`)
+        .then(response => {
+            if (!response.ok) {
```

Comment:
the response will not be okay if there is ParseError right? 

this is preventing your from catching the detailed error message, so you your code on lines 62-64 never get hit.

can you adjust your code so that the alert uses gets the "Invalid address format" from the json if the response is not ok.

Reply:
Ah, good catch! I was wondering why I wasn't getting the desired error back. I will make that change 👍🏽